csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -50,6 +50,7 @@ this.piccache = Array();
 
 const image_base_path = base_static_path + 'interactives/pixel-viewer/img/';
 var source_canvas = document.getElementById('pixel-viewer-interactive-source-canvas');
+var colour_code_rep = 'rgb';
```

Comment:
(nowhere in particular) Funnily enough this doesn't matter too much, as the values will be the same if the squares are the same colour, but if it's easy could we get the hex numbers closer to their characters?
It looks like 00 R rather than R 00 etc
![image](https://user-images.githubusercontent.com/25916475/86544426-d2192500-bf7a-11ea-93da-d53876c62e07.png)